<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelMixer</class>
 <widget class="QWidget" name="ChannelMixer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="BackgroundChMx" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>118</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>118</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="ChMx" name="ch1" native="true">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ChMx" name="ch2" native="true">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ChMx" name="ch3" native="true">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ChMx" name="ch4" native="true">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ChMx" name="ch5" native="true">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ChMx" name="ch6" native="true">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ChMx" name="ch7" native="true">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ChMx" name="ch8" native="true">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ChMx" name="ch9" native="true">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ChMx" name="ch10" native="true">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ChMx" name="ch11" native="true">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ChMx" name="ch12" native="true">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ChMx" name="ch13" native="true">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ChMx" name="ch14" native="true">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ChMx" name="ch15" native="true">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="ChMx" name="ch16" native="true">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>45</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="leftMargin">
      <number>4</number>
     </property>
     <property name="rightMargin">
      <number>4</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>ช่อง:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ChMxComboBox" name="cbCh">
         <property name="minimumSize">
          <size>
           <width>51</width>
           <height>0</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>13</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>4</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>เครื่องดนตรี:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ChMxComboBox" name="cbInts">
         <property name="minimumSize">
          <size>
           <width>201</width>
           <height>0</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="maxVisibleItems">
          <number>14</number>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>P.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="dialPan">
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>กึ่งกลาง</string>
         </property>
         <property name="maximum">
          <number>127</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>64</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>R.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="dialReverb">
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>เสียงก้อง  0</string>
         </property>
         <property name="maximum">
          <number>127</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>C.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="dialChorus">
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="toolTip">
          <string>เสียงประสาน  0</string>
         </property>
         <property name="maximum">
          <number>127</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="chbMuteVoice">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>ปิดเสียงร้อง</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chbLock">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>ล็อก</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSettingVu">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>26</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="toolTip">
        <string>ตั้งค่า LED Meter</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../icons.qrc">
         <normaloff>:/Icons/settings.png</normaloff>:/Icons/settings.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>12</width>
         <height>12</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChMx</class>
   <extends>QWidget</extends>
   <header>ChMx.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BackgroundChMx</class>
   <extends>QFrame</extends>
   <header>ChannelMixer.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ChMxComboBox</class>
   <extends>QComboBox</extends>
   <header>ChMxComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
